From cd120b3976260b7961a939899b6da5184a2ac4da Mon Sep 17 00:00:00 2001
From: Markos Chandras <markos.chandras@imgtec.com>
Date: Tue, 25 Jun 2013 10:29:27 +0000
Subject: [PATCH 019/105] MIPS: Sibyte: Platform: Add load address for CONFIG_SIBYTE_LITTLESUR

Fixes the following build problem:
mips-linux-gnu-ld:arch/mips/kernel/vmlinux.lds:253: syntax error

because VMLINUX_LOAD_ADDRESS was an empty string for that platform
so the vmlinux.lds.S created an invalid section entry on line 50.

Signed-off-by: Markos Chandras <markos.chandras@imgtec.com>
Acked-by: Steven J. Hill <Steven.Hill@imgtec.com>
Cc: sibyte-users@bitmover.com
Cc: linux-mips@linux-mips.org
Patchwork: https://patchwork.linux-mips.org/patch/5548/
Signed-off-by: Ralf Baechle <ralf@linux-mips.org>
(cherry picked from commit 0be0aea2ffa2be1590c86dc226291939bd5535b0)
---
 arch/mips/sibyte/Platform |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/arch/mips/sibyte/Platform b/arch/mips/sibyte/Platform
index d03a075..1bf90dc 100644
--- a/arch/mips/sibyte/Platform
+++ b/arch/mips/sibyte/Platform
@@ -41,3 +41,4 @@ load-$(CONFIG_SIBYTE_RHONE)	:= 0xffffffff80100000
 load-$(CONFIG_SIBYTE_SENTOSA)	:= 0xffffffff80100000
 load-$(CONFIG_SIBYTE_SWARM)	:= 0xffffffff80100000
 load-$(CONFIG_SIBYTE_BIGSUR)	:= 0xffffffff80100000
+load-$(CONFIG_SIBYTE_LITTLESUR) := 0xffffffff80100000
-- 
1.7.1

